README.md

```diff
@@ -55,7 +55,7 @@ Remove all the completed todos after the `Clear completed` button click:
 
 - Implement a solution following the [React task guideline](https://github.com/mate-academy/react_task-guideline#react-tasks-guideline).
 - Use the [React TypeScript cheat sheet](https://mate-academy.github.io/fe-program/js/extra/react-typescript).
-- Replace `<your_account>` with your Github username in the [DEMO LINK](https://<your_account>.github.io/react_todo-app-add-and-delete/) and add it to the PR description.
+- Replace `<your_account>` with your Github username in the [DEMO LINK](https://Liubomyr19/.github.io/react_todo-app-add-and-delete/) and add it to the PR description.
```

Comment:
```suggestion
- Replace `<your_account>` with your Github username in the [DEMO LINK](https://Liubomyr19.github.io/react_todo-app-add-and-delete/) and add it to the PR description.
```
<img width="448" alt="image" src="https://github.com/mate-academy/react_todo-app-add-and-delete/assets/70220667/b6401abb-8bd0-4c32-b624-5d38d5fd522b">
looks like broken link